test/gtest/uct/uct_test.cc

```diff
@@ -213,6 +213,7 @@ uct_test::entity::entity(const resource& resource, uct_iface_config_t *iface_con
     iface_params.tl_name     = const_cast<char*>(resource.tl_name.c_str());
     iface_params.dev_name    = const_cast<char*>(resource.dev_name.c_str());
     iface_params.rx_headroom = rx_headroom;
+    UCS_CPU_ZERO(&iface_params.cpu_set);
```

Comment:
we need this in all other places calling uct_iface_open():
* test/examples/active_message.c
* src/tools/info/tl_info.c
* src/tools/perf/libperf.c